bip-schnorr.mediawiki

```diff
@@ -121,9 +121,9 @@ Input:
 
 The signature is valid if and only if the algorithm below does not fail.
 * Let ''P = point(pk)''; fail if ''point(pk)'' fails.
-* Let ''r = int(sig[0:32])''; fail if ''r &ge; p''.
+* Let ''r = sig[0:32]''; fail if ''int(r) &ge; p''.
 * Let ''s = int(sig[32:64])''; fail if ''s &ge; n''.
-* Let ''e = int(hash(bytes(r) || pk || m)) mod n''.
+* Let ''e = int(hash(r || pk || m)) mod n''.
 * Let ''R = sG - eP''.
 * Fail if ''infinite(R)''.
 * Fail if ''jacobi(y(R)) &ne; 1'' or ''x(R) &ne; r''.
```

Comment:
In commit "leave r as byte array":

If `r` is a byte array it can't be compared with `x(R)`. I'm not sure about this change.

Reply:
Yeah, agree. Dropped the commit.